app/application/views/header.php

```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <meta charset="utf-8">
+        <link rel="icon" type="image/png" href="https://community.dur.ac.uk/dcsgp.08/password/style_updates/app/static/images/favicon-32x32.png" sizes="32x32">
```

Comment:
Please could you switch to using the base_url method, that way it works across installations.
It's used in the rest of the footer for a reference.